### PR TITLE
[release-v1.6] Fix CI

### PR DESCRIPTION
**Pull request by pierDipi**
Some patches in openshift/patches/* were not applied, the issue is that "create-release-branch.sh" doesn't apply them (fixed in https://github.com/openshift-knative/eventing-kafka-broker/pull/451).

Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml [hunk 1]
```diff
@@ -189,6 +189,4 @@ spec:
             capabilities:
               drop:
               - ALL
-            seccompProfile:
```
Comment (matzew) [inline control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml:192]: we still need that for "core" (I know)

File: openshift/release/generate-release.sh [hunk 2]
```diff
@@ -19,11 +19,11 @@ rm -rf $artifacts_dir
 mkdir -p $artifacts_dir
 
 if [ "$release" == "ci" ]; then
-  image_prefix="registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-kafka-"
-  tag=""
+  image_prefix="registry.ci.openshift.org/openshift/knative-eventing-kafka-broker"
```
Comment (matzew) [inline openshift/release/generate-release.sh:22]: did we not update this before on main?

Comment (matzew) [reply openshift/release/generate-release.sh:22]: Or did we now just change the pattern?

OK to do so - just wondering

File: control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml (hunk 1 above)
Comment (pierDipi) [reply control-plane/config/eventing-kafka-broker/200-controller/500-controller.yaml:192]: Yes, I saw your PR

**Pull request merged**